src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/liquidwalk/modes/LiquidWalkVulcan.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * This file is part of LiquidBounce (https://github.com/CCBlueX/LiquidBounce)
+ *
+ * Copyright (c) 2024 CCBlueX
+ *
+ * LiquidBounce is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LiquidBounce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
+ *
+ *
+ */
+package net.ccbluex.liquidbounce.features.module.modules.movement.liquidwalk.modes
+
+import net.ccbluex.liquidbounce.config.Choice
+import net.ccbluex.liquidbounce.config.ChoiceConfigurable
+import net.ccbluex.liquidbounce.event.repeatable
+import net.ccbluex.liquidbounce.features.module.modules.movement.liquidwalk.ModuleLiquidWalk
+
+/*
+    * Bypasses Vulcan 285
+    * Tested on eu.loyisa.cn
+    * Note: May cause occasional step flags if player is in a 1x1 water hole.
+ */
+
+internal object LiquidWalkVulcan : Choice("Vulcan285") {
+
+    private val motion by float("Motion", 0.8f, 0.2f..1.75f)
+
+    override val parent: ChoiceConfigurable
+        get() = ModuleLiquidWalk.modes
+
+    val repeatable = repeatable {
+        if (player.isSubmergedInWater)
+            player.velocity.y = motion.toDouble()
```

Comment:
Have you tested this? I think "isSubmergedInWater" was only true if your head is under water, not the body itself.

Reply:
> Have you tested this? I think "isSubmergedInWater" was only true if your head is under water, not the body itself.

yes i have, you need to be completely underwater.  Vulcan doesn't check for vertical movement if your under water